src/main/js/dashboard-frontend/src/components/details/DetailHeader.tsx

```diff
@@ -96,23 +96,24 @@ export default class DetailHeader extends Component<
   render() {
     return (
       <Header
+        variant={"h1"}
```

Comment:
Just a quick question - was the html tag an h1 before? Are there default styles being applied to this h1?

Reply:
https://cloudscape.design/components/header/?tabId=api#:~:text=false-,variant,-string

Reply:
it would have used whatever the default was, which is h2.